example/lib/main.dart

```diff
@@ -200,7 +192,7 @@ class _DialScreenState extends State<DialScreen> with WidgetsBindingObserver {
 
   @override
   void dispose() {
-    WidgetsBinding.instance!.removeObserver(this);
+    WidgetsBinding.instance.removeObserver(this);
```

Comment:
Let's keep this for now, since `pubspec.yaml` allows >= 2.14, and removing this will cause a null warning.

Incompatible with `2.14`.